package.json

```
@@ -15,11 +15,12 @@
     "prepare:release": "./pipeline/prepareRelease.sh",
     "build": "rm -rf ./dist && tsc",
     "build:watch": "rm -rf ./dist && tsc --watch",
-    "test:unit:watch": "yarn test:unit -- --watch",
+    "test:unit:watch": "yarn test:unit --watch",
     "test:unit": "jest",
     "test": "ts-node ./dockest.ts",
     "test:debug": "ts-node ./dockest.ts debug",
-    "lint": "eslint ./src/**/*.ts"
+    "lint": "eslint ./src/**/*.ts",
```

```suggestion
    "lint": "eslint './src/**/*.ts'",
```

Discovered this a while back that without the single quotes it doesn't pick up on some linting errors 🤷